.github/workflows/benchmarks.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       with:
         toolchain: nightly
     - name: Rebuild benchmarks
-      run: benchmarks-next/build.sh
+      run: benchmarks-next/build-all.sh 5
```

Comment:
At this point, I don't think we need to build all of the benchmarks all the time. The purpose of this is to check that any benchmarks in the directory can indeed be built and by picking 5 of them in each CI run, its probable that we will find any newly-added benchmarks that don't build for some reason.

Reply:
(context: rebuilding the benchmarks is taking ~35 minutes of CI time; the next closest job is running them, which takes ~10 minutes).